cli/internal/terraform/loader.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 //go:embed terraform/*
+//go:embed terraform/*/.terraform.lock.hcl
```

Comment:
I don't think we need to explicitly tell Go to embed this file when we already embed the entire folder

Reply:
I tested this..

> If a pattern names a directory, all files in the subtree rooted at that directory are embedded (recursively), except that files with names beginning with ‘.’ or ‘_’ are excluded. So the variable in the above example is almost equivalent to:
source: https://pkg.go.dev/embed

we could use `//all:go:embed terraform/*` instead. I don't have preference here.

Reply:
Wouldn't use all here. Back then I opted against this since it might accidentally include other trash or hidden files.

If we include hidden files, it should be explicitly imo.